core/src/main/java/com/airbnb/aerosolve/core/features/Features.java

```diff
@@ -29,4 +43,114 @@
     Collections.sort(features);
     return features;
   }
+
+  public Example toExample(boolean isMultiClass) {
```

Comment:
I feel this is not really very generic, for example, we are not taking care of dense feature here. Perhaps just add some 'TODO' or comment here so that people know that what this is doing, what this is not doing.